tools/compile_tf_graph.py

```diff
@@ -568,22 +688,52 @@ def _get_parent_layer(self, layer_name):
       return layer
     assert isinstance(rec_layer, RecStepByStepLayer)
     output = layer.output.copy()
-    output.placeholder = rec_layer.create_state_var(
-      name="base_value_%s" % layer_name, initial_value=output.placeholder, data_shape=output)
+    output = self.make_base_state_vars(layer_name, output)
+    layer = WrappedInternalLayer(name=layer_name, network=self.parent_net, output=output, base_layer=layer)
+    self._parent_layers[layer_name] = layer
+    return layer
+
+  def make_base_state_vars(self, name, output):
+    """
+    make state vars for input from outside the loop (automatic tile_batch)
+    :param str name: layer name
+    :param output: layer output data
+    :rtype: converted output data 
```

Comment:
The format/syntax is wrong. `:rtype:` specifies the return type.

Reply:
You probably mean `:return:` here, which describes what is being returned. But then `:rtype:` is missing.